nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-ldap/src/main/java/org/nuxeo/ecm/directory/ldap/LDAPServerDescriptor.java

```diff
@@ -75,6 +75,9 @@
     @XNode("retries")
     public int retries = 5;
 
+    @XNode("poolingConnectionTimeout")
```

Comment:
This is misnamed or misleading, it's not a connection timeout. Just call it `poolingTimeout`.